core/src/main/java/org/jboss/modcluster/mcmp/impl/DefaultMCMPHandler.java

```diff
@@ -600,7 +600,16 @@ private String sendRequest(MCMPRequest request, Proxy proxy) {
         synchronized (proxy) {
             try {
                 String line = null;
-                String proxyhead = head + proxy.getSocketAddress().getHostName() + ":" + proxy.getSocketAddress().getPort();
+                String hostAddress = proxy.getSocketAddress().getAddress().getHostAddress();
+                String hostName = proxy.getSocketAddress().getHostName();
```

Comment:
InetSocketAddress.getHostName() will perform a reverse DNS lookup.  See:
http://docs.oracle.com/javase/8/docs/api/java/net/InetSocketAddress.html#getHostName--
We should instead use http://docs.oracle.com/javase/8/docs/api/java/net/InetSocketAddress.html#getHostString--